physics/ephemeris_test.cpp

```diff
@@ -497,9 +497,10 @@ TEST_P(EphemerisTest, EarthProbe) {
   }
   // The solution is a line, so the rounding errors dominate.  Different
   // compilers result in different errors and thus different numbers of steps.
-  EXPECT_THAT(probe_positions.size(), AnyOf(Eq(434), Eq(419), Eq(-1)));
+  EXPECT_THAT(probe_positions.size(),
+              AnyOf(Eq(434), Eq(419), Eq(400), Eq(379), Eq(-1)));
```

Comment:
Does this pass on linux? Also perhaps you should remove the old expectations.
What's up with the Eq(-1)? shouldn1t vector::size be nonnegative?